### PR TITLE
Updated the mysql demo to v2

### PR DESCRIPTION
**Pull request by aeroniero33**
https://www.katacoda.com/aeroniero/scenarios/mysql

### CODE REVIEW

File: mysql/000-install-storageos.sh [hunk 1]
```diff
@@ -1,6 +1,69 @@
...
 
-kubectl create -f https://github.com/storageos/cluster-operator/releases/download/1.5.0/storageos-operator.yaml
-kubectl create -f storageos-secret.yaml
-kubectl create -f storageos-cluster.yaml
\ No newline at end of file
+echo "Installing StorageOS and ETCD"
+
+sleep 5
+
+ssh node01 -o StrictHostKeyChecking=no wget https://raw.githubusercontent.com/aeroniero33/tutorials/master/mysql/assets/install-etcd.sh
```
Comment (aeroniero33) [inline mysql/000-install-storageos.sh:8]: I will change this to the storageos repo before merging.

File: mysql/004-failover.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ In order to keep this tutorial simple, we are going to delete the MySQL pod and
 
 Now, you can see the mount was removed but the location is still the same.
```
Comment (avestuk) [inline mysql/004-failover.md:25]: The CLI doesn't show that the volume was detached so I'm not sure what you mean by this.

File: mysql/004-failover.md [hunk 3]
```diff
@@ -2,19 +2,19 @@ Now, let's check that the data was persisted and is accessible on another node.
...
 
-`storageos volume update --label-add storageos.com/replicas=1 $volume`{{execute}}
+`kubectl exec -ti cli -n kube-system -- storageos update volume replicas $volume 1`{{execute}}
 
 Check that the replica was created.
 
-`storageos volume ls --format "table {{.Name}}\t{{.Replicas}}"`{{execute}}
+`kubectl exec -ti cli -n kube-system -- storageos get volume`{{execute}}
 
 > Usually takes ~10 seconds to create the replica in this example.
```
Comment (avestuk) [inline mysql/004-failover.md:20]: This is for the line below but there's no guarantee that the pods are going to be created on two separate nodes unless you use node selectors. That'd add some complexity that may or may not be required. Have a think about what you're trying to illustrate and decide what course to take.

File: mysql/004-failover.md (hunk 2 above)
Comment (aeroniero33) [reply mysql/004-failover.md:25]: This is the output I get: 
image

Is the pod fully deleted in your case? Maybe I should add that in the process.

File: mysql/004-failover.md (hunk 3 above)
Comment (aeroniero33) [reply mysql/004-failover.md:20]: I tried it multiple times and it got deployed to a different node every time, I wonder if the person who wrote this (maybe @Arau ) knew that the scoring system of kube-scheduler would always select the other node. I will try to do it in a clearer/more consistent way though.

File: mysql/004-failover.md [hunk 4]
```diff
@@ -2,19 +2,19 @@ Now, let's check that the data was persisted and is accessible on another node.
 
 Make a note of the location on which the volume got initially prosioned on:
```
Comment (StellaEngle) [inline mysql/004-failover.md:3]: s/prosioned/provisioned

File: mysql/004-failover.md (hunk 3 above)
Comment (aeroniero33) [reply mysql/004-failover.md:20]: Oh wait it does have a node selector actually!

Comment (aeroniero33) [reply mysql/004-failover.md:20]: https://github.com/storageos/tutorials/blob/a38cc8d76159569349a36cec5f0bad526f9df79e/mysql/assets/mysql-pod2.yaml#L25

Comment (StellaEngle) [reply mysql/004-failover.md:20]: Let's chat about it on slack!

**Comment by Arau** [comment]
This scenario is great and enables us to do StorageOS v2 installations really smoothly. We have decided to evolve this PR to fulfil the following

StorageOS persisted volumes vs Non persisted

- StorageOS is bootstrapped already
- Create a Pod without PVC (maybe with volume mount)
- Write to DB for instance MySQL
- Delete the Pod --> Check data is lost

File: mysql/004-failover.md (hunk 2 above)
Comment (avestuk) [reply mysql/004-failover.md:25]: Ah right you are, I think I just clicked through too fast.

**Comment by Arau** [comment]
Sorry, the I put in place a review but I didn't press the button to finish it

**Pull request closed without merging**